pkgs/applications/science/physics/yade/default.nix

```diff
@@ -0,0 +1,121 @@
+with (import <nixpkgs> {});
```

Comment:
please make it a function of the used packages (```{ stdenv, fetchurl, python27Packages etc }:```) like in the .nix files around.
```nixpkgs``` is a bit different than standalone .nix files